File: javascript/26  Remove Duplicates from Sorted Array.js
var removeDuplicates = function(nums) {
    var i = 0;
    var j = 0;
    
    for (l=nums.length; i<l; i++){
        if (nums[j] != nums[i]) {
            nums[++j] = nums[i];
        }
    }
    return j+1;
};

File: C++/26 Remove Duplicates from Sorted Array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        if (nums.size() == 0) {
            return 0;
        }
        
        int j = 1;
        int prev = nums[0];
        
        for (int i = 1; i < nums.size(); i++) {
            if (nums[i] == prev) {
                continue;
            }else {
                nums[j] = nums[i];
                j += 1;
                prev = nums[i];
            }
        }
        return j;
    }
};
